### PR TITLE
Add client wheel url to main spaces 

### PR DESCRIPTION
**Pull request by aliabd**
Adds the missing client wheel url to the requirements for main website spaces.

### CODE REVIEW

File: scripts/upload_website_demos.py [hunk 1]
```diff
@@ -93,13 +95,15 @@ def upload_demo_to_space(
...
     args = parser.parse_args()
     gradio_wheel_url = args.WHEEL_URL + f"gradio-{args.GRADIO_VERSION}-py3-none-any.whl"
+    
     if args.AUTH_TOKEN is not None:
         hello_world_version = str(huggingface_hub.space_info("gradio/hello_world").cardData["sdk_version"])
         for demo in demos:
             if hello_world_version != args.GRADIO_VERSION:
                 upload_demo_to_space(demo_name=demo, space_id="gradio/" + demo, hf_token=args.AUTH_TOKEN, gradio_version=args.GRADIO_VERSION)
-            upload_demo_to_space(demo_name=demo, space_id="gradio/" + demo + "_main", hf_token=args.AUTH_TOKEN, gradio_version=args.GRADIO_VERSION, gradio_wheel_url=gradio_wheel_url)
+            upload_demo_to_space(demo_name=demo, space_id="gradio/" + demo + "_main", hf_token=args.AUTH_TOKEN, gradio_version=args.GRADIO_VERSION, gradio_wheel_url=gradio_wheel_url, gradio_client_url=args.CLIENT_URL)
```
Comment (abidlabs) [inline scripts/upload_website_demos.py:109]: We modify args.WHEEL_URL by adding "gradio-{args.GRADIO_VERSION}-py3-none-any.whl" to it. Do we need to do the same thing for the `gradio_client`?

**Comment by abidlabs** [comment]
Just one question above^ @aliabd but otherwise lgtm

File: scripts/upload_website_demos.py (hunk 1 above)
Comment (aliabd) [reply scripts/upload_website_demos.py:109]: Nope, the gradio_client is just installed from github, so there's no actual wheel file. it's just something like 
`
gradio-client @ git+https://github.com/gradio-app/gradio@c0d25fc1df60987793471b4745ab453bea92dcfe#subdirectory=client/python
`
that's also what we do for the pr demos

Comment (abidlabs) [reply scripts/upload_website_demos.py:109]: Ok great! Thanks for clarifying

**Pull request merged**